src/plugins/foxPage/foxPage.tsx

```diff
@@ -42,24 +48,23 @@ const assetsRoutes: Record<AssetId, FoxPageRoutes> = {
   [FoxyAssetId]: FoxPageRoutes.Foxy,
 }
 
-export type FoxPageProps = {
-  activeAssetId: AssetId
-}
-
-export const FoxPage = (props: FoxPageProps) => {
+export const FoxPage: React.FC<{}> = () => {
   const translate = useTranslate()
   const history = useHistory()
+  const dispatch = useDispatch()
   const assetFox = useAppSelector(state => selectAssetById(state, FoxAssetId))
   const assetFoxy = useAppSelector(state => selectAssetById(state, FoxyAssetId))
 
   const assets = useMemo(() => [assetFox, assetFoxy], [assetFox, assetFoxy])
 
+  const activeAssetId = useRouteAssetId()
   const selectedAssetIndex = useMemo(
-    () => assets.findIndex(asset => asset.assetId === props.activeAssetId),
-    [props.activeAssetId, assets],
+    () => assets.findIndex(asset => asset.assetId === activeAssetId),
+    [activeAssetId, assets],
   )
 
   const selectedAsset = assets[selectedAssetIndex]
+  const foxyMaxTotalSupply = useFoxyMarketData().maxTotalSupply
```

Comment:
```suggestion
  const foxyMarketData = useFoxyMarketData()
  const foxyMaxTotalSupply = foxyMarketData.maxTotalSupply
```
Trying to access a property from a hook like so is going to potentially bring some problems because of components (hooks) lifecycle

Reply:
done